app/views/browse/imagestream.html

```diff
@@ -44,105 +44,14 @@ <h1 class="contains-actions">
-                          <!-- We have no idea why in this case -->
-                          <span ng-if="tag.spec.from">Unresolved</span>
-                        </div>
-                      </div>
-                      <div ng-if="tag.status">
-                        <span ng-if="tag.status.items.length && tag.status.items[0].image" class="nowrap">
-                          <short-id id="{{tag.status.items[0].image | imageName}}"></short-id>
-                          <a href="" ng-if="tag.status.items.length > 1" ng-click="tagShowOlder[tag.name] = !tagShowOlder[tag.name]" ng-attr-title="{{tagShowOlder[tag.name] ? 'Hide Older Images' : 'Show Older Images'}}"><span class="fa fa-clock-o"></span><span class="fa fa-caret-up" ng-if="tagShowOlder[tag.name]" style="margin-left: 3px;"></span></a>
-                        </span>
-                        <span ng-if="!tag.status.items.length"><em>none</em></span>
-                      </div>
-                    </td>
-                    <td data-title="Created">
-                      <span ng-if="tag.status.items.length && tag.status.items[0].image">
-                        <span am-time-ago="tag.status.items[0].created"></span>
-                      </span>
-                    </td>
-                    <td data-title="Pull Spec">
-                      <!-- dockerImageReference values can get long. Use max width and class truncate to elide some of the SHA1 for long values.
-                           They can still be copied even when elided. -->
-                      <div ng-if="tag.status.items.length && tag.status.items[0].dockerImageReference">
-                        <div ng-attr-title="{{tag.status.items[0].dockerImageReference}}" class="td-long-string">
-                          {{tag.status.items[0].dockerImageReference}}
-                        </div>
-                      </div>
-                    </td>
-                  </tr>
-                  <tr ng-repeat="item in tag.status.items" ng-if="tagShowOlder[tag.name] && !$first && item.image">
-                    <td data-title="Tag"><span class="hidden-xs">&nbsp;</span><span class="visible-xs">{{tag.name}}</span></td>
-                    <td class="hidden-xs">&nbsp;</td>
-                    <td data-title="Older Image">
-                      <short-id id="{{item.image | imageName}}"></short-id>
-                    </td>
-                    <td data-title="Created">
-                      <span am-time-ago="item.created"></span>
-                    </td>
-                    <td data-title="Pull Spec">
-                      <div ng-if="item.dockerImageReference" ng-attr-title="{{item.dockerImageReference}}" class="td-long-string">
-                        {{item.dockerImageReference}}
-                      </div>
-                    </td>
-                  </tr>
-                </tbody>
-              </table>
+              <registry-imagestream-body imagestream="imageStream">
```

Comment:
sorry I didn't notice it before, but the switch to use registry-imagestream-body changes the behavior of how annotations are presented to the user, everywhere else in the UI we do not show these to the user unless they click a link to reveal them.  This is because annotations are typically intended for machine use and not end user facing.